### DISCUSSION TITLE
封面更新

### DISCUSSION THREAD

**Discussion by wangrongwei** [wangrongwei/omnetpp_primer#28] [Ideas]
我在考虑是否我们可以更新一下封面，例如使用与仿真相关的、或可以表示这些用途的图片都可。
感觉这个是你的擅长@ruoxu-wang

**Comment by ruoxu-wang** [comment]
这可能需要一些时间

**Comment by ruoxu-wang** [comment]
@wangrongwei 该文档的pdf版本是由markdown转换而来的吗？为何根目录的latex文档还是原始版本？我在考虑将开发环境更换为更常用的TeXstudio和CTEX软件包。

**Reply by wangrongwei** [reply]
这里我没有直接使用latex，主要是由于不便于修改，这里我主要用到了multimarkdown：
`multimarkdown的作用：markdown + latex模板 =》latext=》pdf`
主要还是既想用markdown写文档，又想使用latex的排版。

Anyway，看你决定

**Reply by ruoxu-wang** [reply]
我开一个新branch把内容迁移到latex上吧，你这个操作有点复杂
